doc/source/tokudb/tokudb_variables.rst

```diff
@@ -949,6 +959,32 @@ case there are bugs. Not for use in production.
 Defines direction to be used to perform table scan to check for empty tables
 for bulk loader.
 
+.. variable:: tokudb_enable_fast_update
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Global/Session
+  :dyn: Yes
+  :vartype: Boolean
+  :default: OFF
+
+Toggles the fast updates feature ON/OFF for the ``UPDATE`` statement. Fast
+update involves queries optimization to avoid random reads during their
+execution.
+
+.. variable:: tokudb_enable_fast_upsert
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Global/Session
+  :dyn: Yes
+  :vartype: Boolean
+  :default: OFF
+
+Toggles the fast updates feature ON/OFF for the ``INSERT`` statement. Fast
```

Comment:
Likewise

Reply:
same here, mentioned that fast updates feature is experimental